### ISSUE TITLE
Undefined symbols for architecture armv7

### ISSUE THREAD

**Issue by thd-fox** [Akylas/nativescript-opencv#5]
I needed to update my project to {N}-6.2.0 to run it with latest macOS Catalina.
Then I added your plugin and got the error below:
```Undefined symbols for architecture armv7:
  "cv::String::deallocate()", referenced from:
      cv::String::operator=(cv::String const&) in OpenCVAdditions(OpenCVWrapper.o)
      cv::String::~String() in OpenCVAdditions(OpenCVMat.o)
  "cv::findContours(cv::_InputOutputArray const&, cv::_OutputArray const&, cv::_OutputArray const&, int, int, cv::Point_<int>)", referenced from:
      +[OpenCVWrapper findContours:::::] in OpenCVAdditions(OpenCVWrapper.o)
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using new build systemnote: Planning buildnote: Constructing build description
** ARCHIVE FAILED **´´´

I'd really love to use your plugin with my app. Thanks :)

**Comment by farfromrefug** [comment]
Please give information about version, platform ...
And can you create a simple example project showing the issue?

**Comment by thd-fox** [comment]
Thank you for your fast answer. I'll post as you requested asap.

**Comment by thd-fox** [comment]
Excerpt from commandline:
```
Webpack compilation complete. Watching for file changes.
Webpack build done!
Project successfully prepared (ios)
/Volumes/Data MSD/Kunden/CrediMaxx/CrediMaxx/hooks/after-prepare/nativescript-opencv.js will NOT be executed because it has invalid arguments - platformsData, $usbLiveSyncService.
Building project...´´´

**Comment by thd-fox** [comment]
I created a new project from sidekick with your plugin added only.
When I run `tns run ios` I get the following outputs.

Excerpt from commandline:
```
Webpack compilation complete. Watching for file changes.
Webpack build done!
Project successfully prepared (ios)
/Volumes/Data/AppName/hooks/after-prepare/nativescript-opencv.js will NOT be executed because it has invalid arguments - platformsData, $usbLiveSyncService.
Building project...
```
Later right before the end of the process:

```Merged 775 categories.
Result: 43066 declarations from 144 top level modules
Done! Running time: 7.53487 sec 
Saving metadata generation's stderr stream to: /Volumes/Data/AppName/platforms/ios/build/Debug-iphoneos/metadata-generation-stderr-arm64.txt
/Volumes/Data/AppName/platforms/ios
Undefined symbols for architecture armv7:
  "cv::String::deallocate()", referenced from:
      cv::String::operator=(cv::String const&) in OpenCVAdditions(OpenCVWrapper.o)
      cv::String::~String() in OpenCVAdditions(OpenCVMat.o)
  "cv::findContours(cv::_InputOutputArray const&, cv::_OutputArray const&, cv::_OutputArray const&, int, int, cv::Point_<int>)", referenced from:
      +[OpenCVWrapper findContours:::::] in OpenCVAdditions(OpenCVWrapper.o)
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using new build system
note: Planning build
note: Constructing build description
** ARCHIVE FAILED **

Unable to apply changes on device: XXXXXXXX. Error is: Command xcodebuild failed with exit code 65.
```

`package.json`
```
{
  "nativescript": {
    "id": "org.thd.xxx",
    "tns-android": {
      "version": "6.2.0"
    },
    "tns-ios": {
      "version": "6.2.0"
    }
  },
  "description": "NativeScript Application",
  "license": "SEE LICENSE IN <your-license-filename>",
  "repository": "<fill-your-repository-here>",
  "scripts": {
    "lint": "tslint \"src/**/*.ts\""
  },
  "dependencies": {
    "@angular/animations": "~8.2.0",
    "@angular/common": "~8.2.0",
    "@angular/compiler": "~8.2.0",
    "@angular/core": "~8.2.0",
    "@angular/forms": "~8.2.0",
    "@angular/platform-browser": "~8.2.0",
    "@angular/platform-browser-dynamic": "~8.2.0",
    "@angular/router": "~8.2.0",
    "jstimezonedetect": "~1.0.6",
    "kinvey-nativescript-sdk": "4.2.5",
    "nativescript-angular": "~8.20.0",
    "nativescript-iqkeyboardmanager": "~1.5.0",
    "nativescript-opencv": "0.0.2",
    "nativescript-theme-core": "~1.0.6",
    "nativescript-ui-dataform": "~6.0.0",
    "reflect-metadata": "~0.1.12",
    "rxjs": "^6.4.0",
    "tns-core-modules": "~6.2.0",
    "tns-ios": "^6.2.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~8.2.0",
    "@ngtools/webpack": "~8.2.0",
    "codelyzer": "~4.5.0",
    "nativescript-dev-webpack": "~1.3.0",
    "tns-platform-declarations": "~6.2.0",
    "tslint": "~5.19.0",
    "typescript": "~3.5.3"
  },
  "gitHead": "8567507219abed861fe632ee68b3f1b2a26528dd",
  "readme": "NativeScript Application"
}
```

**Comment by farfromrefug** [comment]
@thd-fox just released 0.0.3 which should fix the issue.
Let me know

**Comment by thd-fox** [comment]
@farfromrefug thank you very much! Now I can run successfully with your plugin!
